packages/cfpb-forms/src/organisms/Multiselect.js

```diff
@@ -172,8 +172,10 @@ function Multiselect( element ) { // eslint-disable-line max-statements
         'class': 'm-form-field m-form-field__checkbox'
       } );
 
+      const _checkboxId = _name + '-' + option.value;
```

Comment:
Maybe we should process `option.value` here and ensure it doesn't contain any spaces. So:

`const checkboxId = _name + '-' + option.value.trim().replace( /\s+/g, '-' ).toLowerCase();`

Reply:
You don't need the leading underscore on `checkboxId`. The leading underscore is for variables defined outside a method that are used in the method, but aren't exported publicly. We're not consistent with this though :-\ as I see `_optionsItemDom` defined above has the underscore, while things like `selectionsItemDom` do not.